pyQuARC/code/datetime_validator.py

```diff
@@ -89,7 +89,7 @@ def validate_datetime_against_granules(datetime, collection_shortname, sort_key,
         Returns:
             (dict) An object with the validity of the check and the instance
         """
-        granules = requests.get(f'{CMR_URL}/search/granules.json?short_name={collection_shortname}&sort_key[]=-{sort_key}').json()
+        granules = requests.get(f'{CMR_URL}/search/granules.json?short_name={collection_shortname}&sort_key[]={sort_key}').json()
```

Comment:
why this change? it should basically do the same thing?

Reply:
Since the validate_beginning_datetime_against_granules function uses validate_datetime_against_granules function. With the "-" before the sort_key (in this case, time_start), it compares the collection's beginning_datetime to the beginning datetime of the last granule.